freetextresponse/freetextresponse.py

```diff
@@ -0,0 +1,437 @@
+'''
+This is the core logic for the FreeText XBlock
+'''
+
+import os
+import re
+
+import pkg_resources
+
+from enum import Enum
+from xblock.core import XBlock
+from xblock.fields import Scope
+from xblock.fields import String
+from xblock.fields import Integer
+from xblock.fields import Float
+from xblock.fragment import Fragment
+
+
+class Freetextresponse(XBlock):
+    #  pylint: disable=too-many-ancestors, too-many-instance-attributes
+    '''
+    Enables instructors to create questions with free-text responses.
+    '''
+    @staticmethod
+    def workbench_scenarios():
+        '''
+        Gather scenarios to be displayed in the workbench
+        '''
+        return [
+            ('FreeText XBlock',
+             '''<sequence_demo>
+                    <freetextresponse />
+                    <freetextresponse name="My First XBlock" />
+                </sequence_demo>
+             '''),
+        ]
+    display_name = String(
+        default='FreeText XBlock',
+        scope=Scope.settings,
+        help="This is the title for this question type",
+    )
+    prompt = String(
+        default='Please enter your response within this text area',
+        scope=Scope.settings,
+        help='This is the prompt students will see when '
```

Comment:
Does it use this help or the one defined in edit.html? I suspect it is using the edit.html one but the xblock docs specify it like this however.